src/controllers/controlpickermenu.cpp

```diff
@@ -415,6 +415,57 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
         hotcueMainMenu->addMenu(hotcueMoreMenu);
     }
 
+    // Intro/outro cues
+    QMenu* introOutroMenu = addSubmenu(tr("Intro / Outro Cues"));
+    QString introStartTitle(tr("Intro Start"));
+    QString introEndTitle(tr("Intro End"));
+    QString outroStartTitle(tr("Outro Start"));
+    QString outroEndTitle(tr("Outro End"));
+    QMenu* introStartMenu = addSubmenu(introStartTitle, introOutroMenu);
+    QMenu* introEndMenu = addSubmenu(introEndTitle, introOutroMenu);
+    QMenu* outroStartMenu = addSubmenu(outroStartTitle, introOutroMenu);
+    QMenu* outroEndMenu = addSubmenu(outroEndTitle, introOutroMenu);
+    QList<QMenu*> introOutroSubmenus;
+    introOutroSubmenus.append(introStartMenu);
+    introOutroSubmenus.append(introEndMenu);
+    introOutroSubmenus.append(outroStartMenu);
+    introOutroSubmenus.append(outroEndMenu);
+    QStringList cueTypeTitles;
+    cueTypeTitles.append(introStartTitle);
+    cueTypeTitles.append(introEndTitle);
+    cueTypeTitles.append(outroStartTitle);
+    cueTypeTitles.append(outroEndTitle);
+    QStringList cueTypeNames;
+    cueTypeNames.append(tr("intro start"));
+    cueTypeNames.append(tr("intro end"));
+    cueTypeNames.append(tr("outro start"));
+    cueTypeNames.append(tr("outro end"));
+    QStringList cueTypeCOs;
+    cueTypeCOs.append("intro_start");
+    cueTypeCOs.append("intro_end");
+    cueTypeCOs.append("outro_start");
+    cueTypeCOs.append("outro_end");
+
+    for (int i = 0; i < introOutroSubmenus.size(); ++i) {
+        addDeckAndSamplerAndPreviewDeckControl(
+                QString("%1_activate").arg(cueTypeCOs[i]),
+                tr("Activate %1 Cue").arg(cueTypeTitles[i]),
```

Comment:
This makes an assumption about sentence structure. I doubt that this will work for all languages. Can we get rid of the for loop and just hardcode all values?

Reply:
hmm. then we'd have 4 sets of almost identical tr strings and way more lines of code. 
we could also use "(cue type): action description."?
I'll check when I'm at my laptop, I think the 'Title, description' displayed in the wizard below the combobox is suffient.

Reply:
I don't know how tr works, but maybe we can substitute *before* passing it to tr?

Reply:
This uses the same structure as we have for hotcues:
`tr("Set Hotcue %1");` hotcue_num
`tr("Set or jump to the %1")` cue_type

I changed the strings to use pretty much the same wording as the hotcue strings.
Ready IMo